venia-integration-tests/package.json

```diff
@@ -5,13 +5,12 @@
     "scripts": {
         "clearCache:mac": "rm -rf ~/Library/Application\\ Support/Cypress/cy/*",
         "test": "cypress open --browser chrome --config-file cypress.config.json",
-        "test:ci": "sh run-tests.sh",
+        "test:ci": "./run-tests",
```

Comment:
I see this script tags parameters, where are you passing those parameters? I don't see any here or in `test:headless`

Reply:
They are run-time parameters and will be sent at runtime by the user.